CRM/Utils/PDF/Label.php

```diff
@@ -185,7 +185,7 @@ public function LabelSetFormat(&$format, $unit) {
    */
   public function generateLabel($text) {
     $args = array(
-      'w' => $this->width,
+      'w' => $this->width - 2 * $this->paddingLeft,
```

Comment:
2 is a magic number here.  Could you please add a comment explaining why this change works?  Thanks!